src/thermal_conductivity/scattering.f90

```diff
@@ -93,8 +94,14 @@ subroutine generate(sr, qp, dr, uc, fct, fcf, opts, tmr, mw, mem)
             call lo_stop_gracefully(['This routine only works with FFT meshes'], lo_exitcode_param, __FILE__, __LINE__)
         end select
 
+        if (opts%seed .gt. 0) then
```

Comment:
we should print it out for bookkeeping purposes

Reply:
also it might be good to check for zeros. It is all a bit annoying. I went with this eventually, not that I am happy with it: https://github.com/tdep-developers/tdep/blob/ccec69a9be1089a8337f8cc5a7f4191777b4e489/src/canonical_configuration/main.f90#L38-L52

Reply:
good point for the check for zeros.
For the printing of the random seed, the integer used (when provided) is printed at the beginning (from the `main.f90`). Do you mean printing the real seed ?

Reply:
no I just overlooked the print statement, see new comment. Let us just check for >= 0 as you do for the print statement instead of checking for 0

Reply:
Done !